Code/GeorgiaLibrarySystem-/Tests/IntegrationTest/StatisticsTest.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using Core;
+using GTLService.Controller;
+using GTLService.DataAccess.Database;
+using GTLService.DataManagement.Database;
+using NUnit.Framework;
+
+namespace Tests.IntegrationTest
+{
+    public class StatisticsTest
+    {
+        private IStatisticsService _statisticsService;
+
+
+        [Test]
+        //Database approach
+        [TestCase("Database", 8, 5)]
+        public void TopTenBooks(string approach, int loanedCopies, int totalCount)
+        {
+            //Arrange
+            Setup(approach);
+
+            //Act
+            var result = _statisticsService.TopTenBooks();
+
+            //Assert
+            Assert.AreEqual(result.Count, totalCount);
+            Assert.AreEqual(result[0].loaned_count, loanedCopies, "in reality it was: " + result[0].loaned_count);
```

Comment:
is this truly correct, I mean your checking 2 different counts in the same test

Reply:
yes